README.md

```diff
@@ -56,7 +56,11 @@ The `test` mode creates bundles that can be used to run the unit and functional
 
 ### Eject
 
-Eject is not currently supported by `cli-build-app`.
+Ejecting `@dojo/cli-build-app` will produce a `config/build-app/ejected.config.js` file. After ejecting, the modes are specified using webpack's `env` flag (e.g., `--env.mode=dev`), defaulting to `dist`. Note that the dojorc can still be used in its existing format to specify build options. You can run build using webpack with:
```

Comment:
You can run build using webpack with: -> You can run a build using webpack with:

Reply:
Thanks, Dylan! I've fixed this.